packages/utils/src/date-utils/conversionUtils.ts

```diff
@@ -7,44 +8,27 @@ export const MILLIS_PER_HOUR = 1000 * 60 * 60;
 export const dayInMs = 86400000;
 export const maxDuration = dayInMs - MILLIS_PER_SECOND;
 
-/**
- * Utility converts milliseconds to a specific unit
- * @param millis
- * @param conversion
- * @returns
- */
-function convertMillis(millis: MaybeNumber, conversion: number): number {
-  if (!millis) {
-    return 0;
-  }
-  return Math.floor(millis / conversion);
-}
-
 /**
  * Converts value in milliseconds to seconds
  * @param millis
  * @returns
  */
-export function millisToSeconds(millis: MaybeNumber): number {
-  return convertMillis(millis, MILLIS_PER_SECOND);
-}
+export function millisToSeconds(
+  millis: MaybeNumber,
+  direction: TimerType.CountDown | TimerType.CountUp = TimerType.CountDown,
+) {
+  if (millis === null) return 0;
+
+  if (direction === TimerType.CountDown) {
+    // 0 + is there to avoid result giving -0
+    return 0 + Math.ceil(millis / MILLIS_PER_SECOND);
+  }
 
-/**
- * Converts value in milliseconds to minutes
- * @param millis
- * @returns
- */
-export function millisToMinutes(millis: MaybeNumber): number {
-  return convertMillis(millis, MILLIS_PER_MINUTE);
-}
+  if (direction === TimerType.CountUp) {
+    return Math.floor(millis / MILLIS_PER_SECOND);
+  }
 
-/**
- * Converts value in milliseconds to hours
- * @param millis
- * @returns
- */
-export function millisToHours(millis: MaybeNumber): number {
-  return convertMillis(millis, MILLIS_PER_HOUR);
```

Comment:
question: dont we need these functions anymore?

Reply:
no could not find a use of it
I can revert this and we kan do a clean-up round later?

Reply:
no, these functions are trivially written so if they are not in use lets bin it